templates/pages/account/orders/details.html

```diff
@@ -54,7 +54,7 @@ <h3 class="account-heading">{{lang 'account.orders.details.order_details'}}</h3>
             {{#if order.shipping_address}}
                 <section class="account-sidebar-block">
                     <h3 class="account-heading">{{lang 'account.orders.details.ship_to'}}</h3>
-                    <ul class="account-order-address">
+                        <ul class="account-order-address">
```

Comment:
Generally looks good. But it seems like `ul` tag was accidentally shifted

Reply:
😨 , fixing that